jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdContainer.java

```diff
@@ -203,7 +211,9 @@ public void start() {
     @Override
     public void close() {
         super.close();
-        deleteDataDirectory(dataDirectory);
+        if(bindVolumn) {
```

Comment:
this check may not be needed as if the data directory os null, then the method won't do anything